src/Microsoft.CodeAnalysis.PerformanceSensitiveAnalyzers/Microsoft.CodeAnalysis.PerformanceSensitiveAnalyzers.md

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
Why is this file being added? We already have a copy here: https://github.com/dotnet/roslyn-analyzers/blob/master/src/PerformanceSensitiveAnalyzers/Microsoft.CodeAnalysis.PerformanceSensitiveAnalyzers.md

Reply:
Also tagging @genlu